src/Pact/Types/Scheme.hs

```diff
@@ -28,20 +29,29 @@ import qualified Pact.JSON.Encode as J
 
 --------- PPKSCHEME DATA TYPE ---------
 
-data PPKScheme = ED25519 | ETH
+data PPKScheme = ED25519 | WebAuthn
```

Comment:
I feel like there are really 4 PRs hiding in this one:

1. Monomorphizing the signature scheme
2. Dropping the ETH PPKScheme
3. Dropping format-address
4. Adding the WebAuthn PPKScheme

I want to hear from @larskuhtz whether we'd like to keep the `ETH` scheme, if we're going to be doing bridging with Ethereum...

Reply:
Just 4? Don't forget bumping base64-bytestring! :) That's got to count as 3 all on its own.

Reply:
I'm leaning toward keeping ETH, see comments elsewhere.